homeassistant/components/gpstracker/__init__.py

```diff
@@ -0,0 +1,53 @@
+"""The GPS Tracker integration."""
+from __future__ import annotations
+
+import gps_tracker
+
+from homeassistant.config_entries import (
+    ConfigEntry,
+    ConfigEntryAuthFailed,
+    ConfigEntryNotReady,
+)
+from homeassistant.const import CONF_ENTITIES, CONF_PASSWORD, CONF_USERNAME, Platform
+from homeassistant.core import HomeAssistant
+
+from .const import CLIENT, DOMAIN
+from .helpers import get_gpstracker_client
+
+PLATFORMS: list[str] = [Platform.DEVICE_TRACKER]
```

Comment:
list[Platform]

Reply:
Fixed in 5467619f0fccbab11588224ced48928e629552f5, thanks!